port/esp32/main/main.c

```diff
@@ -313,112 +311,47 @@ factory_presets_cb_new(size_t device, void *data)
-    "MzhGNkExQkE1Njk1NzEzRDYzLmNydDAlBggrBgEFBQcwAYYZaHR0cDovL3Rlc3Rv\n"
-    "Y3NwLmt5cmlvLmNvbTBaBgNVHR8EUzBRME+gTaBLhklodHRwOi8vdGVzdHBraS5r\n"
-    "eXJpby5jb20vb2NmLzRFNjhFM0ZDRjBGMkU0RjgwQThEMTQzOEY2QTFCQTU2OTU3\n"
-    "MTNENjMuY3JsMAoGCCqGSM49BAMCA0cAMEQCHwXkRYd+u5pOPH544wBmBRJz/b0j\n"
-    "ppvUIHx8IUH0CioCIQDC8CnMVTOC5aIoo5Yg4k7BDDNxbRQoPujYes0OTVGgPA==\n"
-    "-----END CERTIFICATE-----\n";
-
-  const char *root_ca =
-    "-----BEGIN CERTIFICATE-----\n"
-    "MIIB3zCCAYWgAwIBAgIJAPObjMBXKhGyMAoGCCqGSM49BAMCMFMxDDAKBgNVBAoM\n"
-    "A09DRjEiMCAGA1UECwwZS3lyaW8gVGVzdCBJbmZyYXN0cnVjdHVyZTEfMB0GA1UE\n"
-    "AwwWS3lyaW8gVEVTVCBST09UIENBMDAwMjAeFw0xODExMzAxNzMxMDVaFw0yODEx\n"
-    "MjcxNzMxMDVaMFMxDDAKBgNVBAoMA09DRjEiMCAGA1UECwwZS3lyaW8gVGVzdCBJ\n"
-    "bmZyYXN0cnVjdHVyZTEfMB0GA1UEAwwWS3lyaW8gVEVTVCBST09UIENBMDAwMjBZ\n"
-    "MBMGByqGSM49AgEGCCqGSM49AwEHA0IABGt1sU2QhQcK/kflKSF9TCrvKaDckLWd\n"
-    "ZoyvP6z0OrqNdtBscZgVYsSHMQZ1R19wWxsflvNr8bMVW1K3HWMkpsijQjBAMA8G\n"
-    "A1UdEwEB/wQFMAMBAf8wDgYDVR0PAQH/BAQDAgGGMB0GA1UdDgQWBBQoSOTlJ1jZ\n"
-    "CO4JNOSxuz1ZZh/I9TAKBggqhkjOPQQDAgNIADBFAiAlMUwgVeL8d5W4jZdFJ5Zg\n"
-    "clk7XT66LNMfGkExSjU1ngIhANOvTmd32A0kEtIpHbiKA8+RFDCPJWjN4loxrBC7\n"
-    "v0JE\n"
-    "-----END CERTIFICATE-----\n";
-
-  int ee_credid =
-    oc_pki_add_mfg_cert(0, (const unsigned char *)cert, strlen(cert),
-                        (const unsigned char *)key, strlen(key));
-  if (ee_credid < 0) {
-    OC_PRINTF("ERROR installing manufacturer EE cert\n");
-    return;
-  }
-
-  int subca_credid = oc_pki_add_mfg_intermediate_cert(
-    0, ee_credid, (const unsigned char *)inter_ca, strlen(inter_ca));
-
-  if (subca_credid < 0) {
-    OC_PRINTF("ERROR installing intermediate CA cert\n");
-    return;
-  }
-
-  int rootca_credid = oc_pki_add_mfg_trust_anchor(
-    0, (const unsigned char *)root_ca, strlen(root_ca));
-  if (rootca_credid < 0) {
-    OC_PRINTF("ERROR installing root cert\n");
-    return;
-  }
-
-  oc_pki_set_security_profile(0, OC_SP_BLACK, OC_SP_BLACK, ee_credid);
+// #ifndef OC_HAS_FEATURE_PLGD_HAWKBIT
+//   int ee_credid = oc_pki_add_mfg_cert(
+//     0, (const unsigned char *)oc_certificate, strlen(oc_certificate),
+//     (const unsigned char *)oc_certificate_key, strlen(oc_certificate_key));
+//   if (ee_credid < 0) {
+//     OC_PRINTF("ERROR installing manufacturer EE cert\n");
+//     return;
+//   }
+
+//   int subca_credid = oc_pki_add_mfg_intermediate_cert(
+//     device, ee_credid, (const unsigned char *)oc_inter_ca,
+//     strlen(oc_inter_ca));
+
+//   if (subca_credid < 0) {
+//     OC_PRINTF("ERROR installing intermediate CA cert\n");
+//     return;
+//   }
+
+//   int rootca_credid = oc_pki_add_mfg_trust_anchor(
+//     device, (const unsigned char *)oc_root_ca, strlen(oc_root_ca));
+//   if (rootca_credid < 0) {
+//     OC_PRINTF("ERROR installing root cert\n");
+//     return;
+//   }
+
+//   oc_pki_set_security_profile(device, OC_SP_BLACK, OC_SP_BLACK, ee_credid);
```

Comment:
just put to comment about how to set mfg certs to the device.

Reply:
event a comment doesn't seem needed, there are other examples apps that add certificates, the code can be copied from there if needed